src/components/form/dt-tags/dt-tags.js

```diff
@@ -126,6 +129,19 @@ export class DtTags extends DtMultiSelect {
     `;
   }
 
+  _validateRequired() {
+    const { value } = this;
+
+    if (this.required && value) {
```

Comment:
If value is null/undefined and required is true, this won't trigger a required message